server/src/main/kotlin/org/tod87et/roomkn/server/database/Database.kt

```diff
@@ -0,0 +1,17 @@
+package org.tod87et.roomkn.server.database
+
+import org.tod87et.roomkn.server.models.*
+import javax.sql.DataSource
+
+interface Database {
+    fun getRooms(): Result<List<ShortRoomInfo>>
+    fun getRoom(): Result<RoomInfo>
+    fun getRoomReservation(roomId: Int): Result<List<Reservation>>
+    fun createReservation(reserve: UnregisteredReservation): Result<Reservation>
+    fun getUsers(): Result<List<ShortUserInfo>>
+    fun getUser(userId: Int): Result<UserInfo>
+    fun registerUser(user: UnregisteredUserInfo): Result<UserInfo>
+    fun loginUser(user: LoginUserInfo): Result<UserInfo>
+
+    fun clear(): Unit //Clear database for TEST/DEBUG purpose
```

Comment:
Давай без хвостовых комметариев, лучше перед функцией, возможно даже doc-comment'ом
```suggestion
    /** Clear database for TEST/DEBUG purpose */
    fun clear(): Unit
```